io-engine/src/bin/io-engine.rs

```diff
@@ -89,6 +89,11 @@ fn start_tokio_runtime(args: &MayastorCliArgs) {
     if !ENABLE_NEXUS_RESET.load(Ordering::SeqCst) {
         warn!("Nexus reset is disabled");
     }
+    if args.lvm {
+        env::set_var("LVM", "1");
+        env::set_var("LVM_SUPPRESS_FD_WARNINGS", "1");
+        warn!("LVM pool backend is enabled");
```

Comment:
Shouldn't this be an info?
The other messages, like "warn!("Nexus reset is disabled");" are warnings because they are about some functionality _disabled_.

Reply:
The intent of the warn is because lvm support is still experimental, but I can change to info if you think it's better

Reply:
Ah, I see. In that case, shouldn't it explicitly state that it is experimental? "Experimental LVM pool backend is enabled".

Reply:
Done